### PR TITLE
Install operator-sdk

### PR DESCRIPTION
**Pull request by mhmxs**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -212,6 +212,10 @@ MOCKGEN = $(shell pwd)/bin/mockgen
 mockgen:
 	$(call go-get-tool,$(MOCKGEN),github.com/golang/mock/mockgen@latest)
 
+OPERATOR_SDK = $(shell pwd)/bin/operator-sdk
+operator-sdk:
+	$(call curl-get-tool,$(OPERATOR_SDK),https://github.com/operator-framework/operator-sdk/releases/download/v1.7.1/operator-sdk_linux_amd64)
```
Comment (Mojachieee) [inline Makefile:217]: This downloads only the linux version.

Can we add a `[[ $OSTYPE == 'darwin'* ]]` check, so we can download the OSX one

Comment (mhmxs) [reply Makefile:217]: Good idea

Comment (mhmxs) [reply Makefile:217]: @Mojachieee Could you please pick this up? I suggest to replace `linux` to some `$(uname)` magic, but i don`t have macos, so can`t verify the change.

Comment (mhmxs) [reply Makefile:217]: This works on linux: `$(uname  | tr '[:upper:]' '[:lower:]')`

**Review by Mojachieee — APPROVED** [review]
LGTM

**Comment by mhmxs** [comment]
Thanks @Mojachieee

**Pull request merged**